DateTimes/DateTimes/Program.cs

```diff
@@ -0,0 +1,426 @@
+﻿using System;
+using System.Globalization;
+
+namespace DateTimes
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // 1
+            DisplayDate();
+
+            // 2
+            // DisplayModifiedYears();
+
+            //3
+            // FormatDate();
+
+            //4
+            // IncreaseSeconds();
+
+            //5
+            // ShowMillisecons();
+
+            //6
+            // IncreaseMonths();
+
+            //7
+            // GetFirstAndLastSecond();
+
+            //8
+            // GetDifferenceInSeconds();
+
+            //9
+            // DisplayAge();
+
+            //10
+            // Display30Days();
+
+            //11
+            // DaysCounter();
+
+            //12
+            // YesterdayTodayTomorrow();
+
+            //13
+            // Print5Days();
+
+            //14
+            // FirstDayOfWeek();
+
+            //15
+            // GetDays();
+
+            //16
+            GetSundays();
+
+
+        }
+
+        #region 01 Format date according to instructions
+        public static void DisplayDate()
+        {
+            // 1a) 
+            Console.Write("1a) Current date and time: ");
+            Console.WriteLine(DateTime.Now);
+            //  1b)
+            Console.Write("1b) Current year is: ");
+            Console.WriteLine(DateTime.Now.Year);
+            //  1c)
+            Console.Write("1c) Current month is: ");
+            Console.WriteLine((YearMonths)DateTime.Now.Month);
+            //  1d)
+            Console.Write("1d) Current week is: ");
+            Console.WriteLine(WeekOfTheYear());
+            //  1e)
+            Console.Write("1e) Current weekday is: ");
+            Console.WriteLine($"Today is {DateTime.Now.DayOfWeek}");
+            //  1f)
+            Console.Write("1f) The day of the year is: ");
+            Console.WriteLine($"Today is day no. {DateTime.Now.DayOfYear} of year {DateTime.Now.Year}");
+            //  1g)  
+            Console.Write("1f) The day of the month is: ");
+            Console.WriteLine($"Today is day no. {DateTime.Now.Day} of month {DateTime.Now.Month}");
+            //  1h)  
+            Console.Write("1h) The day of week is: ");
+            Console.WriteLine($"Today is day no. {DayOfTheWeek()} of the {WeekOfTheYear()}");
+            // 1j)
+            Console.WriteLine(IsLeapYear());
+            Console.WriteLine("*************************************************************************************");
+        }
+        #endregion
+
+        #region 01d Write a program to display week number of the year
+        public static string WeekOfTheYear()
+        {
+            return ($"Week { CultureInfo.CurrentCulture.Calendar.GetWeekOfYear(DateTime.Now, CalendarWeekRule.FirstDay, DayOfWeek.Monday)}");
+
+        }
+        #endregion
+
+        #region 01h Write a program to display day of week
+        public static int DayOfTheWeek()
```

Comment:
hm...aici trebuia doar sa afisezi: Thursday...etc in functie de ce zi a saptamanii este